templates/includes/header.html

```diff
@@ -48,7 +53,12 @@
                         </li>
                     {% endfor %}
                     <li>
-                        <a href="{% url 'search' %}" class="main-nav-desktop__item">
+                        <a href="{% url 'search' %}" 
+                            class="main-nav-desktop__item
+                            {% if request.path|starts_with:'/search/' %}
+                                main-nav-desktop__item--current-section
+                            {% endif %}
+                        ">
```

Comment:
This bit is a little dodgy. 

@sarahframe / @haydngreatnews Is there a way for me to check against that `{% url 'search' %}` variable? I couldn't figure it out, so I went with a comparing it to a hard-coded string of "/search/".

Note this is only for the pre-/non-JS render of the nav. It gets updated once react mounts.

Reply:
Is it right that the only time search should be highlighted in the nav is when we are on the actual search page, as there won't be any child pages of search?

Reply:
I've pushed a fix for this :) @liamjohnston

Reply:
That's correct. Thanks :)